src/App.jsx

```diff
@@ -67,10 +76,54 @@ class App extends Component {
     });
   }
 
+  GPSTimerOut = () => {
+    clearTimeout(Timer.gameTimer);
+    this.setState({ locationOk: LocationStatus.locationTimerOut });
+    this.restartQuestions();
+  }
+
+  continiusGPSCheck = () => {
```

Comment:
**Suggestion:** I think what you're trying to implement here is monitoring the current position. Currently I get a prompt every once in a while to accept sharing my location. The API exposes a [`watchPosition()`](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API#Watching_the_current_position), which notifies the app whenever the user moves and doesn't prompt them to allow access each time.